File: tests/testdata/codegen/azure-native-v2-eventgrid-pp/python/azure-native-v2-eventgrid.py
import pulumi
import pulumi_azure_native as azure_native

example = azure_native.eventgrid.EventSubscription("example",
    destination={
        "endpoint_type": "EventHub",
        "resource_id": "example",
    },
    expiration_time_utc="example",
    scope="example")

File: tests/testdata/codegen/azure-native-v2-eventgrid-pp/nodejs/azure-native-v2-eventgrid.ts
import * as pulumi from "@pulumi/pulumi";
import * as azure_native from "@pulumi/azure-native";

const example = new azure_native.eventgrid.EventSubscription("example", {
    destination: {
        endpointType: "EventHub",
        resourceId: "example",
    },
    expirationTimeUtc: "example",
    scope: "example",
});
